ogre2/src/Ogre2GpuRays.cc

```diff
@@ -1013,9 +1013,13 @@ void Ogre2GpuRays::Setup1stPass()
   compoChannels.push_back(this->dataPtr->depthTexture);
   compoChannels.push_back(this->dataPtr->particleTexture);
   compoChannels.push_back(this->dataPtr->particleDepthTexture);
+      passScene->mEnableForwardPlus = false;
```

Comment:
Not really sure if we should include this here. @iche033 , there is a script for that, isn't it?

Reply:
yes, let me fix this

Reply:
ok should be good now

Reply:
Thanks!